tayoto_mikaellaantonette/index.html

```diff
@@ -0,0 +1,188 @@
+            />
+            <img
+              src="https://singulart-website-prod.s3.eu-central-1.amazonaws.com/artworks/v2/cropped/17005/main/zoom/1107483_7ba096d0ca55638dc7b87902b64d37e6.jpeg"
+            />
+            <img src="https://static.toiimg.com/photo/108176903.cms" />
+            <img
+              src="https://5.imimg.com/data5/SELLER/Default/2022/6/UQ/ZR/LH/105150937/af1-6-a.jpg"
+            />
+            <img
+              src="https://cdn.shopify.com/s/files/1/0047/4231/6066/t/48/assets/the-kiss-gustav-klimt-1648576625955_800x.jpg"
+            />
+            <img
+              src="https://image.jimcdn.com/app/cms/image/transf/none/path/sa6549607c78f5c11/image/i6b743a5bd2529b6a/version/1500387064/unmissable-paintings-in-europe-la-clairvoyance-by-magritte.jpg"
+            />
+          </div>
+          <hr />
+          <div class="goals">
+            <h2>My Goals</h2>
+            <ol>
+              <li>Finish Studies</li>
+              <li>Get a decent job</li>
+              <li>Buy my own camera</li>
+              <li>Travel around the Philippines</li>
+            </ol>
+          </div>
+          <form class="comment">
+            <div>
+              <h2>Comments</h2>
+              <label for="name">Name: </label>
+              <input type="text" id="name" oninput="name()" />
+            </div>
+            <div>
+              <p>Comment:</p>
+              <textarea
+                name="gcomment"
+                id="gcomment"
+                oninput="comment()"
+                cols="30"
+                rows="10"
+              ></textarea>
+            </div>
+            <div>
+              <button
+                class="submit-goal"
+                type="button"
+                id="comment_button"
+                disabled
+              >
+                Submit
+              </button>
+            </div>
+          </form>
+          <script src="./script.js"></script>
+          <hr />
+          <div>
+            <h2>Comments on my goals</h2>
+            <ul>
+              <li>Goodluck! - Sofia</li>
+              <li>Great Job! - Eliza</li>
+              <li>
+                Focused and embrace the challenges that come your way -Precious
+              </li>
+            </ul>
+            <li>Praying for your journey! - Alona</li>
+            <hr />
+          </div>
+          <div class="contact-form">
+            <div class="contact-header">
+              <h2>Contact Me</h2>
+            </div>
+            <div class="container">
+              <form>
+                <div class="email">
+                  <label for="email">Email </label>
+                  <input type="email" name="email" id="email" />
+                </div>
+                <div class="fname">
+                  <label for="fullname">Full Name</label>
+                  <input type="text" name="fullname" id="fullname" />
```

Comment:
ID should be written in snake case.

Reference:
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5